libparsec/crates/client/tests/unit/certificates_ops/poll.rs

```diff
@@ -0,0 +1,220 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use libparsec_client_connection::{
+    test_register_low_level_send_hook, test_register_send_hook, HeaderMap, ResponseMock, StatusCode,
+};
+use libparsec_protocol::authenticated_cmds;
+use libparsec_tests_fixtures::prelude::*;
+use libparsec_types::prelude::*;
+
+use crate::certificates_ops::PollServerError;
+
+use super::utils::certificates_ops_factory;
+
+#[parsec_test(testbed = "minimal", with_server)]
```

Comment:
Can you avoid using `with_server` ? it's faster and simpler to rely on request mock

Reply:
Done